src/stash/protos/objects.proto

```diff
@@ -64,6 +64,7 @@ message ClusterValue {
     GlobalId linked_object_id = 2;
     RoleId owner_id = 3;
     repeated MzAclItem privileges = 4;
+    ClusterConfig config = 5;
```

Comment:
This probably requires a stash migration to migrate the existing clusters.

Reply:
I'm not sure! In https://github.com/MaterializeInc/materialize/pull/19762/files#diff-ae4537b9c6964f950b962c46de8fa54ca0a55bd2328725b122e6dc2e4d722c3bR271 I default to `Unmanaged` if the proto field isn't set, which should match the current behavior. Let's look into this once the PR is a bit further.

Reply:
Defaulting to `Unmanaged` in this way should be fine. There are maybe two small issues I can see though:

1. Someone changes the default value, which results in external changes even though the serialized data didn't change.
2. Somewhere else (e.g. a migration) we don't consider `None == Unmanaged`.

I don't think either of these are too big of a risk though, so I'm fine with this

Reply:
Added a stash migration.